podvm/Dockerfile.podvm_builder.centos

```diff
@@ -19,12 +19,11 @@ ENV GO_VERSION ${GO_VERSION}
 ENV RUST_VERSION ${RUST_VERSION}
 ENV PROTOC_VERSION ${PROTOC_VERSION}
 
-RUN dnf install -y 'dnf-command(config-manager)' && dnf config-manager --set-enabled powertools && \
-    dnf install -y epel-release epel-next-release && dnf update -y && \
+RUN dnf install -y 'dnf-command(config-manager)' && dnf config-manager --set-enabled crb && \
     dnf groupinstall -y 'Development Tools' && \
-    dnf install -y yum-utils gnupg git curl pkg-config libseccomp-devel gpgme-devel \
+    dnf install -y yum-utils gnupg git pkg-config libseccomp-devel gpgme-devel \
```

Comment:
curl is not needed ?

Reply:
it seems it's already shipped, trying to reinstall it hits a conflict:
```
podman run --rm quay.io/centos/centos:stream9 dnf install curl
CentOS Stream 9 - BaseOS                        2.2 MB/s | 7.9 MB     00:03    
CentOS Stream 9 - AppStream                     4.8 MB/s |  18 MB     00:03    
CentOS Stream 9 - Extras packages               9.3 kB/s |  14 kB     00:01    
Error: 
 Problem: problem with installed package curl-minimal-7.76.1-26.el9.x86_64
  - package curl-minimal-7.76.1-26.el9.x86_64 from @System conflicts with curl provided by curl-7.76.1-26.el9.x86_64 from baseos
  - package curl-minimal-7.76.1-20.el9.x86_64 from baseos conflicts with curl provided by curl-7.76.1-26.el9.x86_64 from baseos
  - package curl-minimal-7.76.1-21.el9.x86_64 from baseos conflicts with curl provided by curl-7.76.1-26.el9.x86_64 from baseos
  - package curl-minimal-7.76.1-22.el9.x86_64 from baseos conflicts with curl provided by curl-7.76.1-26.el9.x86_64 from baseos
  - package curl-minimal-7.76.1-23.el9.x86_64 from baseos conflicts with curl provided by curl-7.76.1-26.el9.x86_64 from baseos
  - package curl-minimal-7.76.1-26.el9.x86_64 from baseos conflicts with curl provided by curl-7.76.1-26.el9.x86_64 from baseos
  - package curl-minimal-7.76.1-12.el9.x86_64 from appstream conflicts with curl provided by curl-7.76.1-26.el9.x86_64 from baseos
  - package curl-minimal-7.76.1-14.el9.x86_64 from appstream conflicts with curl provided by curl-7.76.1-26.el9.x86_64 from baseos
  - cannot install the best candidate for the job
(try to add '--allowerasing' to command line to replace conflicting packages or '--skip-broken' to skip uninstallable packages or '--nobest' to use not only best candidate packages)
```